app/src/main/java/org/mozilla/fenix/components/Core.kt

```diff
@@ -179,10 +181,9 @@ class Core(
      * The [BrowserStore] holds the global [BrowserState].
      */
     val store by lazyMonitored {
-        BrowserStore(
-            middleware = listOf(
+        var middlewareList =
+            listOf(
```

Comment:
nit: We could consider turning this into a `mutableListOf()` and then change the contents of the list instead of creating two different `BrowserStore`, which is a little bit confusing to read at first.

Reply:
Sure, I'll use a mutable list.  Thanks,